run.sh

```diff
@@ -11,13 +11,33 @@
 readonly BINARY_NAME="${__RALLY_INTERNAL_BINARY_NAME}"
 readonly HUMAN_NAME="${__RALLY_INTERNAL_HUMAN_NAME}"
 
+function install_esrally_with_setuptools() {
+    # Check if optional parameter with Rally binary path, points to an existing executable file.
+    if [[ -n $1 ]]; then
+     if [[ -f $1 && -x $1 ]]; then return; fi
```

Comment:
Do we need to enclose the parameter in double quotes to avoid issues with spaces in path names? (i.e. `"${1}"`?)

Reply:
Double quotes are needed only for the backwards compatible `[`. Double square braces do not require [double quoting for the variables](http://mywiki.wooledge.org/BashGuide/TestsAndConditionals#Conditional_Blocks_.28if.2C_test_and_.5B.5B.29).